screens/courses/ViewCourseScreen.js

```diff
@@ -150,7 +206,15 @@ class ViewCourseScreen extends React.Component {
                 })}
               title="Edit Course"
             />
+<<<<<<< HEAD
```

Comment:
there's still a merge conflict here!!!! @loischang how could u

Reply:
O noooo I thought I fixed that one T__T